docs/theme/toc.html

```diff
@@ -17,7 +17,7 @@ <h3 class="side-nav-section-header">
                   <a class="{% if nav_item.active%}is-active{%endif%}" href="{{ nav_item.url|url }}">{{ nav_item.title }}</a>
                 </li>
               </ul>
-              {% if nav_item.active%}
+              <!-- {% if nav_item.active%}
```

Comment:
if this is not needed, I'd rather remove it than comment it.

Reply:
I'm keeping this around as a reference when we expand the navigation again. I'll remove all old stuff when we upgrade to the new design.